pkg/lib/flavor/model/meta.go

```diff
@@ -18,10 +18,10 @@ import (
 type Meta struct {
 	Schema *Schema `json:"schema,omitempty"`
 	// swagger:strfmt uuid
-	ID          uuid.UUID          `json:"id"`
-	Realm       string             `json:"realm,omitempty"`
-	Description Description        `json:"description,omitempty"`
-	Vendor      hcConstants.Vendor `json:"vendor,omitempty"`
+	ID          uuid.UUID              `json:"id"`
+	Realm       string                 `json:"realm,omitempty"`
+	Description map[string]interface{} `json:"description,omitempty"`
```

Comment:
Is there a cleaner way to get the "flavorpart" (i.e., instead of "signedFlavor.Flavor.Meta.Description[flavormodel.FlavorPart].(string)")?

Reply:
Changed the description struct to interface, so that any new keys added in meta section of template can be included. Existing model has flavor parts section inside meta->description. Any change needs to be made to that ?